engines/wintermute/base/gfx/opengl/base_render_opengl3d.cpp

```diff
@@ -582,4 +525,159 @@ bool BaseRenderOpenGL3D::drawSpriteEx(BaseSurfaceOpenGL3D &tex, const Wintermute
 	return true;
 }
 
+void BaseRenderOpenGL3D::renderSceneGeometry(BaseArray<AdWalkplane *> &planes, BaseArray<AdBlock *> &blocks, BaseArray<AdGeneric *> &generics, Camera3D *camera) {
+	_gameRef->_renderer3D->resetModelViewTransform();
+	_gameRef->_renderer3D->setup3D(camera, true);
+
+	glDisable(GL_LIGHTING);
+	glDisable(GL_DEPTH_TEST);
+	glFrontFace(GL_CCW);
+	glPolygonMode(GL_FRONT_AND_BACK, GL_LINE);
+
+	glBindTexture(GL_TEXTURE_2D, 0);
+
+	// render walk planes
+	for (uint i = 0; i < planes.size(); i++) {
+		if (!planes[i]->_active) {
+			continue;
+		}
+
+		planes[i]->_mesh->render();
+
+		//m_Renderer->m_NumPolygons += _planes[i]->m_Mesh->m_NumFaces;
```

Comment:
why commented code?

Reply:
I checked what purpose this member variable actually has. It is only used do display the amount of polygons rendered so far for debugging, so this might just be removed.

Reply:
Comment removed.